cvx/linalg/types.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import numpy.typing as npt
+from typing_extensions import TypeAlias
+
+Matrix: TypeAlias = npt.NDArray[np.float64]
```

Comment:
Two we need to types files or can we reuse one?

Reply:
I would try to avoid a dependency in linalg on markowitz